sdcm/nemesis.py

```diff
@@ -186,6 +186,7 @@ class Nemesis:  # pylint: disable=too-many-instance-attributes,too-many-public-m
     has_steady_run: bool = False    # flag that signal that nemesis should be run with perf tests with steady run
     schema_changes: bool = False
     config_changes: bool = False
+    tablet_phase_1_supported = True  # flag that signal that the nemesis can run on a tablet-enabled cluster
```

Comment:
there are 74 occurrences of `tablet_phase_1_supported = False` in nemesis.py,
can we change the default to False, and enable the relevant ones ?

Reply:
Well, that's what roy asked to do in the comment above.
But I updated the list, now there are much less occurrences